packages/plugins/quorum/README.md

```diff
@@ -0,0 +1,64 @@
+# `embark-quorum`
+
+> Quorum blockchain client plugin for Embark
+
+`embark-quorum` is an Embark plugin that allows for **connecting to* Quorum blockchains. This plugin will not start a Quorum node or nodes automatically as Embark does with other clients.
+
+Using this plugin, you can:
+ - deploy contracts publically and privately to a Quorum network using the Tessera private transaction manager.
+ - send public and private transactions on a Quorum network using the Tessera private transaction manager.
+
+## Setup
+To use the Embark Quorum plugin, you'll need to:
+1. Initialise and run a Quorum network, such as the [7nodes example](https://github.com/jpmorganchase/quorum-examples/tree/master/examples/7nodes).
+2. Install the `embark-quorum` plugin in your dApp.
+3. Configure your dApp's blockchain config.
+4. Configure your dApp's contract config (if necessary).
+
+### Initialise and run a Quorum network
+In order to get a Quorum network up quickly for development, the easiest path is to download the Quorum/Tessera binaries and use them to run the cloned 7nodes example.
+1. Download the latest Quorum binary from https://github.com/jpmorganchase/quorum/releases. Let's say it was downloaded to `/repos/jpmorganchase/quorum/build/bin/geth`.
+2. Update your `PATH` environment variable such that `geth` points to Quorum geth:
+```
+export PATH=/repos/jpmorganchase/quorum/build/bin:$PATH
+```
+3. Download the latest Tessera jar (`tessera-app-0.10.3-app.jar`) from https://github.com/jpmorganchase/tessera/releases. Let's say it was downloaded to 
+4. Update your `TESSERA_JAR` environment variable to point to the downloaded Tessera jar:
+```
+export TESSERA_JAR=/Users/emizzle/Code/__Github/jpmorganchase/tessera/tessera-dist/tessera-app/target/tessera-app-0.10.3-app.jar
+```
+5. Clone the 7nodes example repo:
+```
+git clone https://github.com/jpmorganchase/quorum-examples
+cd examples/7nodes
+```
+6. Initialise Quorum using the desired consensus (`istanbul`, `raft`, `clique`):
+```
+# ./{consensus}-init.sh
+./istanbul-init.sh
+```
+7. Start Quorum using the desired consensus from step 6:
+```
+# ./{consensus}-start.sh tessera
+./istanbul-start.sh
+```
+  - NOTE: the default 7nodes example does not start Quorum geth with WebSockets. We can get around this by updating `./istanbul-start.sh` before executing it, as per https://github.com/jpmorganchase/quorum-examples/pull/221. As an alternative, clone [the fork with these changes](https://github.com/emizzle/quorum-examples) (on the `7nodes-websocket-support` branch) in Step 5 instead.
+
+After a few moments, you should have 7 Quorum nodes up and running.
+
+To stop the network:
+```
+./stop.sh
+```
+
+### Install the `embark-quorum` plugin in your dApp
```

Comment:
More documentation is coming! Just wanted to get the rest of the changes out first so you guys could have a chance to review.